megamek/src/megamek/common/verifier/TestInfantry.java

```diff
@@ -384,6 +384,9 @@ public static double getWeightExact(Infantry infantry, CalculationReport report)
 
         InfantryMount mount = infantry.getMount();
         int activeTroopers = infantry.getInternal(Infantry.LOC_INFANTRY);
```

Comment:
It feels like making this
`int activeTroopers = Math.max(0, infantry.getInternal(Infantry.LOC_INFANTRY));`
would do it more concisely

Reply:
> It feels like making this `int activeTroopers = Math.max(0, infantry.getInternal(Infantry.LOC_INFANTRY));` would do it more concisely

I agree @SJuliez , that is a much more concise way to do it. Changed, thanks!